packages/menus/src/elements/Menu.spec.js

```diff
@@ -42,10 +42,11 @@ describe('Menu', () => {
   let wrapper;
   let onChangeSpy;
 
-  const basicExample = ({ onChange } = {}) => (
+  const basicExample = ({ onChange, maxHeight } = {}) => (
     <Menu
       data-test-id="menu"
       onChange={onChange}
+      maxHeight={maxHeight}
```

Comment:
You've added the prop here but aren't actually testing it?

Reply:
Weird, my tests didn't get added. Yay rebasing.

I've now updated with some tests that should get coveralls to stop complaining.